### DISCUSSION TITLE
multi-tenant when using Azure SSO

### DISCUSSION THREAD

**Discussion by dgilmore82** [Velocidex/velociraptor#3193] [Q&A] [answered]
Does Velociraptor need to be configured as a multi-tenant application?
 
We have deployed Velociraptor on a ECS cluster as a Docker image that gets spun up as a container behind a Application load balancer. When we changed the server.config.yaml file to use SSO, the page we get after using our domain creds said that the application needed to be configured as a multi-tenant application. When we deployed Velociraptor on-prem (version 0.6.5), we didn't have to do anything other then add the client info and tenant. I noticed in version 0.6.6 the [multi-tenant-mode](https://docs.velociraptor.app/blog/2022/2022-08-15-release-notes/#multi-tenant-mode) was implemented. 

We are currently on version 0.7.0-3, is there anything we need to do to the server config file to allow us to successfully authenticate to Velociraptor via Azure SSO?

 

Was there

**Comment by scudette** [comment, marked as answer]
Velociraptor multi org feature is unrelated to Azure in any way so the same SSO deployment process should be used

**Reply by dgilmore82** [reply]
Understood, Thank you and happy new year!!
